Library/Homebrew/api.rb

```diff
@@ -37,8 +35,8 @@ def fetch(endpoint)
       raise ArgumentError, "Invalid JSON file: #{Tty.underline}#{api_url}#{Tty.reset}"
     end
 
-    sig { params(endpoint: String, target: Pathname).returns(Hash) }
-    def fetch_json_api_file(endpoint, target:)
+    sig { params(endpoint: String, target: Pathname).returns(T.any(Array, Hash)) }
```

Comment:
I usually use `T.untyped` for methods that return parsed JSON (as does [sorbet itself](https://github.com/sorbet/sorbet/blob/master/rbi/stdlib/json.rbi)), but was able to get tests to pass with a slight extension of the existing `sig` here.